Header header

int32 id
float32 nx
float32 ny
float32 nz
float32 d 
